source/renderer/app/components/wallet/receive/Address.js

```diff
@@ -5,110 +5,44 @@ import CopyToClipboard from 'react-copy-to-clipboard';
 import classnames from 'classnames';
 import SVGInline from 'react-svg-inline';
 import styles from './Address.scss';
-import iconQR from '../../../assets/images/qr-code.inline.svg';
 import iconCopy from '../../../assets/images/clipboard-ic.inline.svg';
+// import type { ByronWalletAddress as ByronWalletAddressType } from '../../../api/addresses/types';
```

Comment:
Can this be removed or uncomment? Is it waiting for an API implementation?

Reply:
@daniloprates I probably didn't saw that. It is not needed anymore